### PR TITLE
1d physics

### PR DESCRIPTION
**Pull request by andytorrestb**
adding 1D physics for simple docking to master branch. Please run and confirm that nothing is broken as part of the review process.

### CODE REVIEW

**Comment by andytorrestb** [comment]
needs fixes

File: pyrpod/MissionPlanner.py [hunk 1]
```diff
@@ -311,7 +312,32 @@ def calc_delta_m(self, dv, isp):
...
+
+            Returns
+            -------
+            dm : float
+                Change in mass calculated using the ideal rocket equation.
+        """
+        a = (dv/v_e)
+        m_f = self.vv.mass
+        dm = m_f * (np.exp(a) - 1)
+        # self.vv.mass += dm
```
Comment (nickpalumbo8) [inline pyrpod/MissionPlanner.py:339]: I have changes to calc_delta_mass_v_e on fuel_calc

File: pyrpod/MissionPlanner.py [hunk 2]
```diff
@@ -609,4 +635,7 @@ def plot_thrust_envelope(self):
 
             fig.savefig("test" + str(firing[0]) + ".png")
             plt.show()
-        return
\ No newline at end of file
+        return
+
+    def calc_dv(self, v_e, m_o, m_f):
```
Comment (nickpalumbo8) [inline pyrpod/MissionPlanner.py:640]: I have a calc_delta_v with different parameters that is called when calculating propellant expenditure in the JFH

File: tests/test_case_20.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
+# Andy Torres
+# University of Central Florida
+# Department of Mechanical and Aerospace Engineering
+# Last Changed: 2-3-24
+
+# ========================
+# PyRPOD: test/test_case_20.py
```
Comment (nickpalumbo8) [inline tests/test_case_20.py:7]: 20 is taken

File: tests/test_case_21.py [hunk 4]
```diff
@@ -0,0 +1,41 @@
+# Andy Torres
+# University of Central Florida
+# Department of Mechanical and Aerospace Engineering
+# Last Changed: 12-14-23
+
+# ========================
+# PyRPOD: test/test_case_13.py
```
Comment (nickpalumbo8) [inline tests/test_case_21.py:7]: 21 is taken

**Review by nickpalumbo8 — APPROVED** [review]

**Pull request closed without merging**